src/Microsoft.AspNetCore.Mvc.Core/ProducesAttribute.cs

```diff
@@ -87,7 +87,19 @@ public override void OnResultExecuting(ResultExecutingContext context)
             {
                 // Check if there are any IFormatFilter in the pipeline, and if any of them is active. If there is one,
                 // do not override the content type value.
-                if (context.Filters.OfType<IFormatFilter>().All(f => f.GetFormat(context) == null))
+                var filtersInactive = true;
```

Comment:
This translates to `filter's Inactive`. A better name for this would be `isFilterActive`.